test/specs/scenarios/crossDevice.js

```diff
@@ -16,6 +16,7 @@ const options = {
     'CrossDeviceMobileConnected',
     'CrossDeviceSubmit',
     'VerificationComplete',
+    'SelfieIntro',
```

Comment:
This and `selfieIntro` at line 38 are unused here. I think this is one of the ESlint errors caught by the failed CI build.

Reply:
removed